pyproject.toml

```diff
@@ -59,10 +59,26 @@ exclude_lines = [
 ]
 
 [tool.pyright]
-include = ["examples", "hikari"]
+include = ["hikari", "examples"]
+exclude = ["examples/simple_dashboard.py", "**/__init__.py", "hikari/internal/enums.py", "hikari/internal/fast_protocol.py"]
```

Comment:
Why do the init's, enum and fast protocol need to be ignored?

And for simple dashboard, wouldnt it be better to just ignore rillrate there instead?

Reply:
`__init__.py` must be ignored because there will be some unused variables when importing directly.

`enums.py` and `fast_protocol.py` have a dynamically generated namespace. Mypy already ignores these files and uses stubs instead. 
I'd love to fix all of the errors in those files but that is kinda outside the scope of this PR and should be resolved separately.

I doubt there is any way to ignore specifically rillrate. If there is it'd be very helpful in other places too I think.